### PR TITLE
Feature/controllers

### PR DESCRIPTION
**Pull request by tkz96**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:id])
+    @posts = @user.posts
+  end
+
+  def show
+    @posts = @user.posts.find(params[:id])
+  end
+end
```
Comment (VICTOR-CHUKWU) [inline app/controllers/posts_controller.rb:10]: - [ ] There should be a corresponding view file for each controller, kindly add the files to your view folder

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @user = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+end
```
Comment (VICTOR-CHUKWU) [inline app/controllers/users_controller.rb:9]: - [ ]  There should be a corresponding view file for each controller, kindly add the files in your view folder

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @tkz96  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb [hunk 3]
```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:id", to: "posts#show"
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
```
Comment (gemmen29) [inline config/routes.rb:9]: - [ ] [OPTIONAL] To make your routes cleaner. Kindly write the routes in that way
```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
 end
```

**Review by gemmen29 — APPROVED** [review]
Hi @tkz96, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**